pyega3/libs/data_file.py

```diff
@@ -252,7 +252,7 @@ def print_local_file_info_genomic_range(prefix_str, file, gr_args):
             f" referenceMD5={gr_args[1]}, start={gr_args[2]}, end={gr_args[3]}, format={gr_args[4]})"
         )
 
-    def download_file_retry(self, num_connections, output_file, genomic_range_args, max_retries, retry_wait,
```

Comment:
The user no longer can specify the name of the output file. It will always be the `displayFileName`.